### PR TITLE
chore(release): changelog 0.13

### PR DESCRIPTION
**Pull request by Duologic**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,81 @@
 # Changelog
 
+## 0.13 (2020-12-1?)
+
+#### :building_construction: Inline environments
+
+One of the most debated features of the past months has landed. It is now
+possible to define Tanka Environments inline. It is now possible to leverage all
```
Comment (sh0rez) [inline CHANGELOG.md:8]: "It is now possible" twice

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,81 @@
...
+
+Tanka :heart: Jsonnet, and so do you. With this release, you can now access the
+`JSONNET_PATH` that Tanka uses to find all libraries. Try something this in your
+environment:
+
+```console
+$ JSONNET_PATH=$(tk tool jpath environments/prometheus) jsonnet-lint environments/prometheus/main.jsonnet
+```
+
+### Features
```
Comment (sh0rez) [inline CHANGELOG.md:39]: Several PRs below are from non-Grafana contributors. We gave them a shout-out by putting their name behind their PR in other Changelogs, let's do that here as well

**Review by sh0rez — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Duologic) [reply CHANGELOG.md:8]: noice

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,91 @@
...
+`JSONNET_PATH` that Tanka uses to find all libraries. Try something this in your
+environment:
+
+```console
+$ JSONNET_PATH=$(tk tool jpath environments/prometheus) jsonnet-lint environments/prometheus/main.jsonnet
+```
+
+#### :speech_balloon: Github Discussions
+
+The Tanka project trying out GitHub Discussions as the primary support channel:
```
Comment (sh0rez) [inline CHANGELOG.md:41]:
```suggestion
The Tanka project is trying out GitHub Discussions as the primary support channel:
```

**Pull request merged**